tools/flow_schema/main.go

```diff
@@ -0,0 +1,172 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"flag"
+	"os"
+	"path/filepath"
+	"runtime"
+	"text/template"
+)
+
+const flowSchemaTemplate = `# Schema generated by "tools/flow_schema.go" DO NOT EDIT.
+# To generate run 'make generate'
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta2
```

Comment:
What do you think about migration between Kubernetes versions? The v1beta2 API will be deprecated, but this manifest is with hardcoded API versions.

Instead of generating a whole file, the better option is to generate only the list of namespaces and pass it to a module template.  Yet there is still an issue with external modules.

Reply:
Added helm condition to check kubernetes version.